aim/api/resource.py

```diff
@@ -1496,3 +1496,285 @@ def __init__(self, **kwargs):
+
+
+class InfraRspanVsrcGroup(AciResourceBase):
+    """A source relation to all eps with traffic that will be spanned.
+
+    Identity attribute is The virtual source end point group policy name
+    and bundled ports group name.
+    """
+
+    identity_attributes = t.identity(
+        ('acc_bndle_grp_name', t.name),
+        ('name', t.name))
+    other_attributes = t.other(
+        ('monitored', t.bool))
+
+    _aci_mo_name = 'infraRsSpanVSrcGrp'
+    _tree_parent = InfraAccBundleGroup
+
+    def __init__(self, **kwargs):
+        super(InfraRspanVsrcGroup, self).__init__({'monitored': False},
+                                                  **kwargs)
+
+
+class InfraRspanVsrcApGroup(AciResourceBase):
+    """A source relation to all eps with traffic that will be spanned.
+
+    Identity attribute is The virtual source end point group policy name
+    and access port group name.
+    """
+
+    identity_attributes = t.identity(
+        ('acc_port_grp_name', t.name),
+        ('name', t.name))
+    other_attributes = t.other(
+        ('monitored', t.bool))
+
+    _aci_mo_name = 'infraRsSpanVSrcGrp__ap'
+    _tree_parent = InfraAccPortGroup
+
+    def __init__(self, **kwargs):
+        super(InfraRspanVsrcApGroup, self).__init__({'monitored': False},
+                                                    **kwargs)
+
+
+class InfraRspanVdestGroup(AciResourceBase):
+    """A source relation to all eps to which the SPAN packets will be spanned.
+
+    Identity attribute is The virtual source end point group policy name
+    and bundled ports group name.
+    """
+
+    identity_attributes = t.identity(
+        ('acc_bndle_grp_name', t.name),
+        ('name', t.name))
+    other_attributes = t.other(
+        ('monitored', t.bool))
+
+    _aci_mo_name = 'infraRsSpanVDestGrp'
+    _tree_parent = InfraAccBundleGroup
+
+    def __init__(self, **kwargs):
+        super(InfraRspanVdestGroup, self).__init__({'monitored': False},
+                                                   **kwargs)
+
+
+class InfraRspanVdestApGroup(AciResourceBase):
+    """A source relation to all eps to which the SPAN packets will be spanned.
+
+    Identity attribute is The virtual source end point group policy name
+    and access port group name.
+    """
+
+    identity_attributes = t.identity(
+        ('acc_port_grp_name', t.name),
+        ('name', t.name))
+    other_attributes = t.other(
+        ('monitored', t.bool))
+
+    _aci_mo_name = 'infraRsSpanVDestGrp__ap'
+    _tree_parent = InfraAccPortGroup
+
+    def __init__(self, **kwargs):
+        super(InfraRspanVdestApGroup, self).__init__({'monitored': False},
+                                                     **kwargs)
+
+
+class SpanSpanlbl(AciResourceBase):
+    """Resource representing SPAN src-dest group match label.
+
+    Identity attributes is name of SPAN destination group.
+    """
+
+    identity_attributes = t.identity(
+        ('vsg_name', t.name),
+        ('name', t.name))
```

Comment:
According to https://pubhub.devnetcloud.com/media/apic-mim-ref-501/docs/MO-spanSpanLbl.html, this is missing the tag.